Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -3,16 +3,29 @@ import Foundation
 extension Snapshotting where Value == URLRequest, Format == String {
   /// A snapshot strategy for comparing requests based on raw equality.
   public static let raw = SimplySnapshotting.lines.pullback { (request: URLRequest) in
-
     let method = "\(request.httpMethod ?? "GET") \(request.url?.absoluteString ?? "(null)")"
 
     let headers = (request.allHTTPHeaderFields ?? [:])
       .map { key, value in "\(key): \(value)" }
       .sorted()
 
-    let body = request.httpBody
-      .map { ["\n\(String(decoding: $0, as: UTF8.self))"] }
-      ?? []
+    let body: [String]
+    do {
+      if #available(iOS 11.0, *) {
```

Comment:
Aha, needs `macOS 10.3` and `tvOS 11.0` in here.

Reply:
Oof good catch. Will get to next time I have internet!

Reply:
I gotchu